nextcord/embeds.py

```diff
@@ -360,7 +365,11 @@ def footer(self) -> _EmbedFooterProxy:
         return EmbedProxy(getattr(self, "_footer", {}))  # type: ignore
 
     def set_footer(
-        self: E, *, text: MaybeEmpty[Any] = EmptyEmbed, icon_url: MaybeEmpty[Any] = EmptyEmbed
+        self: E,
+        *,
+        text: MaybeEmpty[Any] = EmptyEmbed,
+        icon_url: MaybeEmpty[Any] = EmptyEmbed,
+        icon_file: MaybeEmpty[File] = EmptyEmbed,
```

Comment:
I don't think so as 607 added support for items where they were cast to bytes originally, however, we directly send the files using `http.send_files` which takes `Sequence[File]`. Admittedly this could be reworked, but I feel it is outside the scope of this pull request. What do you think?